pgctl/cli.py

```diff
@@ -410,7 +410,10 @@ def debug(self):
                 'Must debug exactly one service, not: ' + commafy(self.service_names),
             )
 
-        self.stop()
+        if service.state['state'] != 'down':
+            self.stop()
```

Comment:
It's not useless if you're issuing a stop directly. It's confusing and unnecessary if you're attempting to debug a stopped service.